### PR TITLE
upgrade travis and realm for xcode 11

### LINKED ISSUE #1201: Make compatible with latest xcode

### PR DESCRIPTION
**Pull request by James-Sangalli**
Closes https://github.com/AlphaWallet/alpha-wallet-ios/issues/1737

### CODE REVIEW

File: AlphaWallet/Tokens/Coordinators/TokensCoordinator.swift [hunk 1]
```diff
@@ -147,6 +147,7 @@ class TokensCoordinator: Coordinator {
         controller.navigationItem.leftBarButtonItem = UIBarButtonItem(title: R.string.localizable.cancel(), style: .plain, target: self, action: #selector(dismiss))
         let nav = UINavigationController(rootViewController: controller)
         nav.modalPresentationStyle = .formSheet
+        navigationController.modalPresentationStyle = .fullScreen
```
Comment (hboon) [inline AlphaWallet/Tokens/Coordinators/TokensCoordinator.swift]: This should be `nav` instead.

File: AlphaWallet/Tokens/Coordinators/TokensCoordinator.swift [hunk 2]
```diff
@@ -164,6 +165,7 @@ class TokensCoordinator: Coordinator {
         coordinator.delegate = self
         coordinator.start()
         addCoordinator(coordinator)
+        viewController.modalPresentationStyle = .fullScreen
```
Comment (hboon) [inline AlphaWallet/Tokens/Coordinators/TokensCoordinator.swift]: Should this be for the new UINavigationController instance below instead?

File: AlphaWallet/Tokens/Coordinators/TokensCoordinator.swift [hunk 3]
```diff
@@ -164,7 +165,9 @@ class TokensCoordinator: Coordinator {
         coordinator.delegate = self
         coordinator.start()
         addCoordinator(coordinator)
-        viewController.present(UINavigationController(rootViewController: coordinator.serversViewController), animated: true)
+        let uiCoordinator = UINavigationController(rootViewController: coordinator.serversViewController)
```
Comment (hboon) [inline AlphaWallet/Tokens/Coordinators/TokensCoordinator.swift]: Name `uiCoordinator` is misleading.

**Comment by James-Sangalli** [comment]
@hboon there are further issues with xcode 11 which need to be addressed, namely #1745 & the following UI screenshots: 

Screen Shot 2020-02-29 at 1 25 31 pm
Screen Shot 2020-02-29 at 1 25 46 pm
Screen Shot 2020-02-29 at 1 26 06 pm
Screen Shot 2020-02-29 at 1 26 31 pm
Screen Shot 2020-02-29 at 1 26 45 pm
Screen Shot 2020-02-29 at 1 26 58 pm

**Comment by hboon** [comment]
@James-Sangalli good for review

Let's see if tests passes. Travis config is made to run against iOS simulator 12.4.

**Comment by James-Sangalli** [comment]
@hboon testing

**Comment by James-Sangalli** [comment]
@hboon seems good but just one thing. I noticed the top bar looks like this: 

photo_2020-03-06_10-25-15

**Comment by hboon** [comment]
> seems good but just one thing. I noticed the top bar looks like this:

@James-Sangalli with big title? It is supposed to be like this. Was already like this before the project upgrade.

Travis keeps failing when installing pods. I've tried to restart it again to see if it helps. Doesn't seem to be our fault. But wouldn't be good if Travis can't run this config.

**Comment by hboon** [comment]
Hold off merging.

Not expecting code changes, but might juggle commits a bit and rebase to get Travis to work.

**Comment by hboon** [comment]
@James-Sangalli OK. Have cleaned it up. Good for review and merge again.

**Comment by hboon** [comment]
This helped make Travis work and should give us better performance when doing `pod install` locally: https://blog.cocoapods.org/CocoaPods-1.7.2/

**Comment by hboon** [comment]
Spawns a couple of issues to follow up:

* #1746 
* #1748 
* #1749

**Comment by hboon** [comment]
> @hboon seems good but just one thing. I noticed the top bar looks like this:
> 
Seems ok now with:

**Comment by hboon** [comment]
Sigh. Hold again please. One of the pods might be causing a crash - specifically when entering password while backing Keystore JSON file

**Comment by hboon** [comment]
@James-Sangalli all good again. (I hope)

**Pull request merged**